SwiftCart/Model/Services/Remote/NetworkManager.swift

```diff
@@ -35,5 +35,105 @@ class NetworkManager : Networking {
                 return Observable.just(data)
             }
     }
-}
+    
+    
```

Comment:
create a generic function to use post or get or delete or put and don't forget to add it's header in the protocol